resources/scripts/quick_start.sh

```diff
@@ -52,40 +53,53 @@ print_message "" "" ""
 print_message "" "    Let's find out if your system has what it takes to run Warnet..." ""
 print_message "" "" ""
 
-minikube_path=$(command -v minikube || true)
-if [ -n "$minikube_path" ]; then
-    print_partial_message " ⭐️ Found " "minikube" ": $minikube_path " "$BOLD"
-else
-    print_partial_message " 💥 Could not find " "minikube" ". Please follow this link to install it..." "$BOLD"
-    print_message "" "   https://minikube.sigs.k8s.io/docs/start/" "$BOLD"
-    exit 127
-fi
-
-kubectl_path=$(command -v kubectl || true)
-if [ -n "$kubectl_path" ]; then
-    print_partial_message " ⭐️ Found " "kubectl" ": $kubectl_path " "$BOLD"
-else
-    print_partial_message " 💥 Could not find " "kubectl" ". Please follow this link to install it..." "$BOLD"
-    print_message "" "   https://kubernetes.io/docs/tasks/tools/" "$BOLD"
-    exit 127
-fi
-
 docker_path=$(command -v docker || true)
 if [ -n "$docker_path" ]; then
     print_partial_message " ⭐️ Found " "docker" ": $docker_path" "$BOLD"
 else
     print_partial_message " 💥 Could not find " "docker" ". Please follow this link to install Docker Engine..." "$BOLD"
     print_message "" "   https://docs.docker.com/engine/install/" "$BOLD"
-    exit 127
+    ERROR_CODE=127
 fi
 
 current_user=$(whoami)
+current_context=$(docker context show)
```

Comment:
I'm getting an error here

```
--> docker context show
default
```

Reply:
What happens when you run `uname`?

Is the error that it "could not find <your user> in the docker group"?